atomic_defi_design/qml/Constants/Tutorials.qml

```diff
@@ -0,0 +1,31 @@
+pragma Singleton
+import QtQuick 2.15
+
+QtObject {
+    readonly property var tutorial_list: {
+        "portfolio": portfolio_help
+    }
+	readonly property var portfolio_help: [
+		{name: qsTr(`WELCOME TO ATOMICDEX`),
+        description: qsTr(`AtomicDEX Desktop is revolutionary cryptocurrency wallet where you can store your coins and tokens and make peer to peer trades using <a href="https://komodoplatform.com/">Komodo Platform's</a> <a href"https://developers.komodoplatform.com/basic-docs/atomicdex/atomicdex-api.html">atomic swap protocol</a>.An overview of your portfolio is available in the "Dashboard" tab, with a summary of your hodlings fiat price, fiat value and overall percentage. The "Wallet" tab  offers additional detail for each enabled coin, including transaction history, receiving address (in text and QR code format) and "send to" form for making payments. The "Addressbook" tab lets you save destination addresses for quick and convenient selection when making payments. The "DEX" tab is what makes AtomicDEX special. Here, you can create buy or sell orders for any enabled coin pair, or review and select existing trades listed in the orderbook. Being a DEX, there is no middle man - you keep sole control of your keys! Click here more info on <a href="https://cryptoadventure.org/beginners-guide-to-atomic-swaps-solving-the-centralization-problem/">how atomic swaps work</a> `),

```

Comment:
Each time you use AtomicDEX Desktop here, you want to use the app name, such as done in the FAQ

Reply:
```cpp
engine.rootContext()->setContextProperty("atomic_app_name", QString{DEX_NAME});
    engine.rootContext()->setContextProperty("atomic_app_website_url", QString{DEX_WEBSITE_URL});
    engine.rootContext()->setContextProperty("atomic_app_support_url", QString{DEX_SUPPORT_URL});
    engine.rootContext()->setContextProperty("atomic_app_discord_url", QString{DEX_DISCORD_URL});
    engine.rootContext()->setContextProperty("atomic_app_twitter_url", QString{DEX_TWITTER_URL});
    engine.rootContext()->setContextProperty("atomic_app_primary_coin", QString{DEX_PRIMARY_COIN});
    engine.rootContext()->setContextProperty("atomic_app_secondary_coin", QString{DEX_SECOND_PRIMARY_COIN});
    ```